### PR TITLE
Adding unsafe, FIXME and TODO rules into code style section

### PR DESCRIPTION
**Pull request by vsbogd**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,5 +34,24 @@ Feel free to raise draft PR if you need an advice or help with your changes.
...
 ## Code style
 
-We have no specific code style rules for now. Please take a look at the
-existing code and stick to its style.
+We have small set of code style rules for now. The rule of thumb is to take a look
+at the existing code and stick to its style.
+
+### General
+
+If you want to leave some reminder in code, for example to fix somthing later,
```
Comment (noskill) [inline CONTRIBUTING.md:42]:
```suggestion
If you want to leave some reminder in code, for example to fix something later,
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -34,5 +34,24 @@ Feel free to raise draft PR if you need an advice or help with your changes.
...
+which should be done before the PR is merged. Add a comment starting with `TODO`
+to mark the improvement which can be postponed and done later by a separate PR.
+Main purpose of `TODO` comment is to trigger a developer who looks at the code
+after you and make him fix the issue if it is apropriate. If change or question
+is big enough or it affects the API of the module it is better to raise an issue
+instead.
+
+### Rust
+
+When working on Rust C API prefere making `unsafe` blocks as small as possible.
```
Comment (noskill) [inline CONTRIBUTING.md:53]:
```suggestion
When working on Rust C API prefer making `unsafe` blocks as small as possible.
```

Comment (vsbogd) [reply CONTRIBUTING.md:53]: Thanks!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -34,5 +34,24 @@ Feel free to raise draft PR if you need an advice or help with your changes.
...
+It allows easily detect blocks which can be a source of issues. Usually it is
```
Comment (noskill) [inline CONTRIBUTING.md:54]:
```suggestion
This makes it easier to find blocks that might be source of issues.
```

**Review by ngeiswei — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -34,5 +34,24 @@ Feel free to raise draft PR if you need an advice or help with your changes.
...
+at the existing code and stick to its style.
+
+### General
+
+If you want to leave some reminder in code, for example to fix something later,
+you can do it by two ways. Add a comment starting with `FIXME` to mark something
+which should be done before the PR is merged. Add a comment starting with `TODO`
+to mark the improvement which can be postponed and done later by a separate PR.
+Main purpose of `TODO` comment is to trigger a developer who looks at the code
+after you and make him fix the issue if it is apropriate. If change or question
```
Comment (ngeiswei) [inline CONTRIBUTING.md:47]:
```suggestion
after you and make him fix the issue if it is appropriate. If the change or question
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -34,5 +34,24 @@ Feel free to raise draft PR if you need an advice or help with your changes.
...
+is big enough or it affects the API of the module it is better to raise an issue
+instead.
```
[the remark this answers is not in the document]
Comment (vsbogd) [reply CONTRIBUTING.md]: Thanks, Nil!

**Pull request merged**